zipline/testing/fixtures.py

```diff
@@ -224,18 +272,36 @@ class WithAssetFinder(object):
     zipline.testing.make_future_info
     zipline.testing.make_commodity_future_info
     """
+    ASSET_FINDER_EQUITY_SIDS = tuple(map(ord, 'ABC'))
+    ASSET_FINDER_EQUITY_SYMBOLS = None
+    ASSET_FINDER_EQUITY_START_DATE = alias('START_DATE')
+
+    @classproperty
+    def ASSET_FINDER_EQUITY_END_DATE(cls):
+        # add one day to end to make sure the asset exists through the end
+        # of the data
+        return cls.END_DATE + pd.Timedelta(days=1)
```

Comment:
I don't think assets are available on their end date, I think it is `[start, end)`. I could be wrong in which case I will remove the extra day.